index.html

```diff
@@ -138,6 +137,59 @@ <h2 class="heading-secondary">Easy. Simple. Functional!</h2>
         </div>
       </section>
 
+      <section class="testimonial">
+        <span class="subheading marginbottom-sm">Testimonials</span>
+        <h2 class="heading-secondary">What people say about us?</h2>
+
+        <div class="testimonial-content">
+
+          <div class="testimonial__card">
+            <p class="testimonial__text">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corrupti doloremque accusamus provident. Placeat enim libero repellat, incidunt id delectus ipsum quam quos tempore nihil eius, vel laudantium voluptas dignissimos nulla.</p>
+            <div class="testimonial__details">
+              <img class="testimonial__details--photo" src="img/testimonial_photo1.jpg" alt="Photo of a costumer">
+              <h3 class="heading-tertiary testimonial__details--name">Amelia Georgiana</h3>
+              <div class="flex testimonial__details--star">
+                <span class="iconify" data-icon="entypo:star" style="color: #ff8a00;" data-width="20" data-height="20"></span>
```

Comment:
Remove inline and use a class instead,  it will make the code easy to maintain.
Could you also add a class to start that is not filled, like when people choose 4/5 starts.